composables/use-logger.ts

```diff
@@ -50,8 +50,12 @@ export function useLogEvent(eventName: string, eventParams: EventParams = {}) {
     console.error(`Failed to track event with Meta Pixel: ${eventName}`, eventParams)
   }
 
-  const { instance: crisp } = useScriptCrisp()
+  const config = useRuntimeConfig()
+  const crispId = String(config.public.scripts.crisp.id)
+
+  const { instance: crisp } = useScriptCrisp({ id: crispId })
   if (crisp) {
+    crisp.do('chat:hide')
```

Comment:
Move `const { onLoaded: listenCrispLoaded } = useScriptCrisp()` to a plugin to do this globally instead?